requirements.txt

```diff
@@ -1,3 +1,4 @@
+craft-store
```

Comment:
I feel really dumb but I only just noticed that 'pip install craft-store' and 'pip install craft_store' (with an underscore) both work. For my education: How does this work?

Reply:
My first response was ¯\_(ツ)_/¯ but I did some Googling and it looks like PyPI normalizes names so that underscores are automatically converted to dashes. I would assume that `craft_store` was the original name of the package, which got normalized to `craft-store`, and `pip` is smart enough to normalize `craft_store` to `craft-store` when it installs a package.